Intersect.Client.Framework/Entities/IResource.cs

```diff
@@ -1,11 +1,8 @@
 using Intersect.Framework.Core.GameObjects.Resources;
-using Intersect.GameObjects;
 
 namespace Intersect.Client.Framework.Entities;
 
 public interface IResource : IEntity
 {
     ResourceDescriptor? Descriptor { get; }
-    
-    bool IsDepleted { get; }
```

Comment:
Is there a reason why this was removed?

Reply:
IsDepleted return IsDead, so i'm return IsDead directly